src/pages/api/addItem.ts

```diff
@@ -13,86 +13,54 @@ export default async function handler(
   if (request.method === "POST") {
     try {
```

Comment:
please try handling all the errors without nested try/catch blocks, handle all the errors in the parent catch block